ecosystem/indexer-grpc/indexer-grpc-data-service/src/service.rs

```diff
@@ -365,7 +416,7 @@ impl RawData for RawDataServerWrapper {
                             end_version = end_of_batch_version,
```

Comment:
Rename current_version to start_version?

Reply:
done.